docs/sdk/api/sdkClient.md

```diff
@@ -69,6 +69,7 @@ Returns a `Promise` with the accumulated result of each `processFn` calls.
 - `options` _(Object)_
   - `accumulate` _(Boolean)_: (default `true`) a flag to indicate whether all the results of the iterations should be accumulated. This is useful if you want to e.g. fetch all the entities of an API endpoint and do something with it at the end. _Be careful that this might lead to memory problems if the fetched data gets too big. If it's not necessary to have all the data when the process function resolves, it's recommended to disable this option_.
   - `total` _(Number)_: a number to indicate the total amount of items to be fetched from all API calls
+  - `disableSort` _(Boolean)_: (default `false`) disables sorting during iteration, useful when deleting multiple entries. Please do not use it otherwise as it will introduce an infinite loop.
```

Comment:
I don't think this is correct - Mongo will have some sorting internally that is consistent, unless the underlying data changes. The sorting provided by Mongo may or may not be a sorting by `id`.

The more likely result is that you'll miss elements, not that you'll end in an infinite loop (as the `lastId` should increase in every iteration)?

Reply:
Ahh, very good question. Here I was trying to point out that with the PR's implementation it also won't use `lastId` but keep fetching until there's no element left to fetch. 

So the infinite loop could come from the library side - in the case where the user is not deleting every element it's getting.

Does that make sense?

Reply:
I already said to Daniel directly I don't think this is a good solution.

The best way would be to implement a distinct loop method which uses the proposed way by deleting till the result count is below the limit parameter. Even a graphql query could be used to just retrieve the id & version to save bandwidth